tests/test_frame_compare.py

```
@@ -232,6 +232,182 @@ def boom(*_args: object, **_kwargs: object) -> list[object]:
+        "correlation": 0.95,
+        "target_fps": 24.0,
+        "status": "auto",
+    }
+
+    monkeypatch.setattr(
+        frame_compare.audio_alignment,
+        "load_offsets",
+        lambda _path: (reference_path.name, {target_path.name: dict(cached_entry)}),
+    )
+    def _fail_measure(*_args: object, **_kwargs: object) -> list[AlignmentMeasurement]:
+        raise AssertionError("measure_offsets should not run")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)